iac-extensions/docker/src/main/resources/org/sonar/l10n/docker/rules/docker/S7021.html

```diff
@@ -0,0 +1,27 @@
+<p><code>WORKDIR</code> instruction should be used with absolute path for clarity and reliability.</p>
```

Comment:
IMO `instructions` and adding an `an` before absolute path would sound a bit better

Reply:
Good catch! Here is the [PR](https://github.com/SonarSource/rspec/pull/4091) on RSPEC.